include/aspect/simulator.h

```diff
@@ -404,6 +404,19 @@ namespace aspect
        */
       void compute_current_constraints ();
 
+      /**
+       * Compute the reference viscosity for the current time by
```

Comment:
this comments confused me for a second. Maybe start with "This function updates the pressure_scaling variable ..."